python/paddle/fluid/compiler.py

```diff
@@ -480,3 +486,184 @@ def _get_places(self, place, place_list):
                 place_list = cpu_places()
         assert place_list, "No places for execution."
         return place_list
+
+
+class IpuCompiledProgram(object):
+    """
+    :api_attr: Static Graph
+
+    The IpuCompiledProgram is used to transform a program to a ipu-target program.
+
+    Args:
+        program(framework.Program): This argument is the Program being executed.
+        scope: This argument is the scope which contains model parameters.
+        ipu_strategy: This argument is used to build the program with the
+            specified options, such as training or inference mode, batch size in
+	    popart, dtype, etc.
+
```

Comment:
ipu_strategy(IpuStrategy): This argument is used to build the program with the
            specified options, such as training or inference mode, batch size in
	    popart, dtype, etc. please refer to :code:`paddle.static.IpuStrategy`.

Reply:
OK